recipe/build.sh

```diff
@@ -67,6 +62,22 @@ cmake --build . \
     --target magma_sparse \
     --verbose
 
+case "$target_platform" in
+    linux-64)
+        STRIP_CMD="x86_64-conda-linux-gnu-strip"
+        ;;
+    linux-aarch64)
+        STRIP_CMD="aarch64-conda-linux-gnu-strip"
+        ;;
+    linux-ppc64le)
+        STRIP_CMD="powerpc64le-conda-linux-gnu-strip"
+        ;;
+    *)
+        X="unknown"
+        ;;
+esac
+$BUILD_PREFIX/bin/$STRIP_CMD ./lib/libmagma_sparse.so
```

Comment:
Is STRIP broken or something?
![image](https://github.com/user-attachments/assets/a7c56264-fd81-4a7f-9f3e-87ed7697ba2d)

it seems exported at least for x86_64, but i might be missing something...

Reply:
Ah, I forgot we had an environment variable for that.